src/__tests__/Nav.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Navbar from '../components/Nav';
+
+it('renders correctly', () => {
+  const navbar = renderer
+    .create(<Router><Navbar /></Router>)
+    .toJSON();
+  expect(navbar).toMatchSnapshot();
+});
```

Comment:
As requested by the previous reviewer please add unit tests for pure functions in your Redux code and integration tests for your application using the React Testing Library.

This means:

- Add unit tests for several components of the app in your case the `City.js`, `Cities.js`, and `CitiesList.js` components as well.
- Add unit tests for some pure functions in Redux code (these are your action creators and reducers). 
- You can perform a mocking data fetching and Redux Store connection to add integration tests using React Testing Library.